src/NUnitTestAdapter/NUnitEngine/DiscoveryConverter.cs

```diff
@@ -0,0 +1,389 @@
+﻿// ***********************************************************************
+// Copyright (c) 2020-2020 Charlie Poole, Terje Sandstrom
+//
+// Permission is hereby granted, free of charge, to any person obtaining
+// a copy of this software and associated documentation files (the
+// "Software"), to deal in the Software without restriction, including
+// without limitation the rights to use, copy, modify, merge, publish,
+// distribute, sublicense, and/or sell copies of the Software, and to
+// permit persons to whom the Software is furnished to do so, subject to
+// the following conditions:
+//
+// The above copyright notice and this permission notice shall be
+// included in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+// EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+// LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+// WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+// ***********************************************************************
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Xml;
+using System.Xml.Linq;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel;
+using NUnit.VisualStudio.TestAdapter.Internal;
+
+
+
+namespace NUnit.VisualStudio.TestAdapter.NUnitEngine
+{
+    public class DiscoveryConverter
+    {
+        internal static class NUnitXmlAttributeNames
```

Comment:
When I suggested making this class `internal`, I was thinking `DiscoveryConverter` was only one of two classes with copies of these constants. I can't find the second place now, so I hope I didn't imagine that. Sorry!

Reply:
There was another class, but it was removed as the refactoring went forward.  There are more temporary classes here I expect to go away later.